### PR TITLE
dev-util/clion: version bump to 2017.3.4

### PR DESCRIPTION
**Pull request by strahlc**
* dev-util/clion: version bump to 2017.3.4
* dev-util/clion: remove old version

### CODE REVIEW

File: dev-util/clion/clion-2017.3.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
```
Comment (mgorny) [inline dev-util/clion/clion-2017.3.4.ebuild]: Please use a single `inherit` with multiple eclasses.

File: dev-util/clion/clion-2017.3.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+inherit eutils
+
+SLOT="0"
```
Comment (mgorny) [inline dev-util/clion/clion-2017.3.4.ebuild]: Please keep variables in standard order:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

RDEPEND
```

File: dev-util/clion/clion-2017.3.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	insinto "${dir}"
+	doins -r *
+	fperms 755 "${dir}"/bin/{clion.sh,fsnotifier{,64}}
+
+	make_wrapper "${PN}" "${dir}/bin/${PN}.sh"
+	newicon "bin/${PN}.svg" "${PN}.svg"
+	make_desktop_entry "${PN}" "clion" "${PN}" "Development;IDE;"
+
+	# recommended by: https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit
+	mkdir -p "${D}/etc/sysctl.d/" || die
```
Comment (mgorny) [inline dev-util/clion/clion-2017.3.4.ebuild]: Use `dodir` here, that's its exact purpose.

File: dev-util/clion/clion-2017.3.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	echo "fs.inotify.max_user_watches = 524288" > "${D}/etc/sysctl.d/30-idea-inotify-watches.conf" || die
```
Comment (mgorny) [inline dev-util/clion/clion-2017.3.4.ebuild]: This kind of system defaults belong in `/usr/lib/sysctl.d`.

File: dev-util/clion/clion-2017.3.4.ebuild (hunk 1 above)
Comment (strahlc) [reply dev-util/clion/clion-2017.3.4.ebuild]: done

File: dev-util/clion/clion-2017.3.4.ebuild (hunk 2 above)
Comment (strahlc) [reply dev-util/clion/clion-2017.3.4.ebuild]: done

File: dev-util/clion/clion-2017.3.4.ebuild (hunk 3 above)
Comment (strahlc) [reply dev-util/clion/clion-2017.3.4.ebuild]: done

File: dev-util/clion/clion-2017.3.4.ebuild (hunk 4 above)
Comment (strahlc) [reply dev-util/clion/clion-2017.3.4.ebuild]: done

File: dev-util/clion/clion-2017.3.4.ebuild [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	insinto "${dir}"
+	doins -r *
+	fperms 755 "${dir}"/bin/{clion.sh,fsnotifier{,64}}
+
+	make_wrapper "${PN}" "${dir}/bin/${PN}.sh"
+	newicon "bin/${PN}.svg" "${PN}.svg"
+	make_desktop_entry "${PN}" "clion" "${PN}" "Development;IDE;"
+
+	# recommended by: https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit
+	dodir "/usr/lib/sysctl.d/" || die
```
Comment (Zlogene) [inline dev-util/clion/clion-2017.3.4.ebuild]: you do not need `|| die` here as well as quotes for its target

Comment (strahlc) [reply dev-util/clion/clion-2017.3.4.ebuild]: done

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**